lib/constants.js

```diff
@@ -20,6 +22,10 @@ const PLATFORM_URL = LOCALDEV ? new URL('pear/', swapURL) : new URL('../../../',
 const PLATFORM_DIR = toPath(PLATFORM_URL)
 const PLATFORM_LOCK = toPath(new URL('corestores/platform/primary-key', PLATFORM_URL))
 
+const buf = b4a.allocUnsafe(32)
+sodium.crypto_generichash(buf, b4a.from(PLATFORM_DIR))
+const PIPE_ID = b4a.toString(buf, 'hex')
```

Comment:
i would define a helper in the bottom of the file hashToHex to make this a little clearer. then we can also "save" the compute on none windows, ie

```
const PIPE_ID = isWindows ? hashToHex(...) : null
```